### PR TITLE
For intra fix

### PR DESCRIPTION
**Pull request by kowey**

### CODE REVIEW

File: attelo/parser/intra.py [hunk 1]
```diff
@@ -30,6 +30,56 @@ class IntraInterPair(namedtuple("IntraInterPair",
     pass
 
 
+def for_intra(dpack, target):
+    '''
```
Comment (moreymat) [inline attelo/parser/intra.py]: As a minor formatting issue, there are two conventions from PEP0257 that could be enforced here for a low cost:
* docstrings are enclosed by triple double quotes,
* multi-line docstrings begin with a summary line.

Comment (kowey) [inline attelo/parser/intra.py]: Yeah, that's worth fixing.  Should really be spotted automatically.  Never trust a man to do a machine's job.

File: attelo/parser/tests.py [hunk 2]
```diff
@@ -103,3 +108,56 @@ def test_postlabel_parser(self):
...
+
+class IntraTest(unittest.TestCase):
+    """Intrasentential parser"""
+    def test_for_intra(self):
+        'test that sentence roots are identified correctly'
+        # pylint: disable=invalid-name
+        a1 = EDU('a1', 'a', 0, 1, 'a', 's1')
+        a2 = EDU('a2', 'b', 2, 3, 'a', 's1')
+        a3 = EDU('a3', 'c', 4, 5, 'a', 's1')
+        b1 = EDU('b1', 'a', 0, 1, 'a', 's2')
```
Comment (moreymat) [inline attelo/parser/tests.py]: a1 and b1 have the same start and end, same for a2/b2, a3/b3.

File: attelo/parser/tests.py [hunk 3]
```diff
@@ -103,3 +108,56 @@ def test_postlabel_parser(self):
...
+                                        (b3, b1),
+                                        (b3, b2)],
+                              data=scipy.sparse.csr_matrix([[1], [1], [1],
+                                                            [1], [1], [1],
+                                                            [1], [1], [1],
+                                                            [1], [1], [1],
+                                                            [1], [1], [1],
+                                                            [1], [1], [1]]),
+                              target=np.array([3, 1, 1, 4, 1, 4, 1, 1, 1,
+                                               1, 1, 1, 4, 1, 4, 1, 1, 1]),
```
Comment (moreymat) [inline attelo/parser/tests.py]: Are you sure these are the right target integers? 0- or 1-based?

**Comment by moreymat** [comment]
Once the above issues are addressed, we can merge.

**Pull request merged**